module.json

```diff
@@ -35,7 +35,12 @@
     "ualloc": "^1.0.0",
     "minar": "^1.0.0",
     "core-util": "^1.0.0",
-    "compiler-polyfill": "^1.1.0"
+    "compiler-polyfill": "^1.1.0",
+    "greentea-client": "^0.1.4"
+  },
+  "testDependencies": {
+    "unity": "^2.0.0",
+    "greentea-client": "^0.1.4"
```

Comment:
For code coverage functions in retarget: [here](https://github.com/ARMmbed/mbed-drivers/blob/devel_transport/source/retarget.cpp#L188) and [here](https://github.com/ARMmbed/mbed-drivers/blob/devel_transport/source/retarget.cpp#L237).